src/components/form/LogOutButton.js

```diff
@@ -0,0 +1,9 @@
+const LogOutButton = ({onClicked, children}) => (
```

Comment:
This component is rendered by the ListPane component, so I wonder if it should be relocated to the list file.

Reply:
The `LogOutButton` component is rendered in the ListPane component at this point because we don't have a `layout/Header` component, yet, where it ordinarily and more properly belongs, so I shoved it in the `form` folder temporarily with other form elements pending implementation of further details, which might determine other location(s) to situate the log-out button.

The `AddTodoForm` component tentatively sits in the `form` folder for the same reason because, rather than unilaterally edit or remove the existing counterpart now in the `list` folder (which is not reusable), I thought it more desirable for people to compare and see how it, and the `form/InputWithLabel` component, are reusable and more robust so to be desirable to replace the counterparts in the `list` folder in the progress to come. The `form/InputWithLabel` has been used in `auth/LoginForm` and `auth/SignUpForm` and is suitable for any other elements in the project that require form input--it can also be modified according to the theme of the UI to what we like and continues to maintain uniformity in the central scheme of the app.

The non-reusable nature of `list/InputWithLabel` should be obvious. The major shortfall is that it'd require focus for every instance in which it is implemented--i.e., all input fields in a form would be concurrently highlighted.

Further, the `form/AddTodoForm` can simply be renamed `AddTitleForm` and reused anywhere in the app where a single input filed with submit button is required for different purposes, whereas the `list/AddTodoForm` is a fixture with insertion of data id that will be redundant (all such will be assigned by the database) and that is through improper means (i.e., `Date.now()`).